examples/wo_cellarea.yaml

```
@@ -0,0 +1,84 @@
+general:
+  name: "wo_cellarea"
+  cmor_version: "CMIP6"
+  mip: "CMIP"
+  CMIP_Tables_Dir: "/work/ab0995/a270243/pymorize/cmip6-cmor-tables/Tables"
+  CV_Dir: "/work/ab0995/a270243/pymorize/cmip6-cmor-tables/CMIP6_CVs"
+pymorize:
+  # parallel: True
+  warn_on_no_rule: False
+  use_flox: True
+  dask_cluster: "slurm"
+  dask_cluster_scaling_mode: fixed
+  fixed_jobs: 12
+rules:
+  - name: cellarea_wo
+    inputs:
+      - path: /work/ab0995/a270243/pymorize_sample_data
+        pattern: wo_fesom_.*nc
+    cmor_variable: wmo
+    model_variable: wo
+    grid_file: /pool/data/AWICM/FESOM1/MESHES/core/griddes.nc
+    mesh_path: /pool/data/AWICM/FESOM1/MESHES/core
+    output_directory: .
+    variant_label: r1i1p1f1
+    experiment_id: piControl
+    source_id: AWI-CM-1-1-HR
+    model_component: seaIce
+    grid_label: gn
+    pipelines:
+      - default
+pipelines:
+  - name: default
+    steps:
+      - "pymorize.gather_inputs.load_mfdataset"
+      - "pymorize.generic.get_variable"
+      - "script:///work/ab0995/a270243/pymorize/contrib/wo_cellarea.py:nodes_to_levels"
+      - "pymorize.timeaverage.compute_average"
+      - "script:///work/ab0995/a270243/pymorize/contrib/wo_cellarea.py:cellarea"
```

Names need to be consistent with my suggestion above:
```suggestion
      - "script:///work/ab0995/a270243/pymorize/contrib/wo_cellarea.py:weight_by_cellarea_and_density"
```